### DISCUSSION TITLE
手写甲骨文的识别模型如何下载

### DISCUSSION THREAD

**Discussion by henriettaknight** [PaddlePaddle/PaddleOCR#14071] [Q&A] [answered]
我最近在研究如何识别手写的甲骨文，发现社区已经有通过Paddleocr完成的类似的工作，论文见[链接](https://aistudio.baidu.com/projectdetail/5216041?contributionType=1)。想参照论文中的步骤复现看一下对甲骨文的识别效果，但是论文中给的模型下载链接已经失效，如图。
![image](https://github.com/user-attachments/assets/59aaf3de-f6e6-4cee-b857-3af6694d7a95)

请问哪里可以下载到该模型，衷心的感谢~

**Comment by GreatV** [comment]
https://paddlepaddle.github.io/PaddleOCR/latest/ppocr/blog/ocr_book.html#_2

**Reply by henriettaknight** [reply, marked as answer]
链接可以下载，谢谢
